spec/controllers/api/v1/users_controller_spec.rb

```diff
@@ -5,9 +5,29 @@
     request.accept = 'application/json'
   end
 
+  describe '#show' do
+    before do
+      @user = create :user_with_team, :chris
+    end
+
+    it 'returns user data' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.